### ISSUE TITLE
Outdated instructions and screenshots on 'Verifying your custom domain for GitHub Pages' article

### ISSUE THREAD

**Issue by hey-august** [github/docs#32322]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/)

### What part(s) of the article would you like to see updated?

The **Verifying your custom domain for GitHub Pages** article appears to incorrectly describe an older version of the GitHub repository Settings UI, particularly the Pages tab.

### [User Site Instructions](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/verifying-your-custom-domain-for-github-pages#verifying-a-domain-for-your-user-site)

- Step 2 describes a "Code, planning, and automation" section of the sidebar. This sidebar section is now just `Code and automation`.
- Step 3 reads: "On the right, click Add a domain." This button no longer exists. This step should instead direct the user to enter the custom domain in the text box.
- Step 4 references an old UI.
- Step 5 includes a [screenshot](https://docs.github.com/assets/cb-168509/mw-1440/images/help/pages/verify-dns.webp) from an older version of the Pages Settings UI. The section is titled "Add a DNS TXT record". I was unable to locate this section in the Pages tab of my repository Settings.

### In summary:
The custom domain verifications instructions are out of date.
It's unclear if custom domain verification is still supported.

### Additional information

This issue references the **User Site** instructions on this page. The same issue may affect the **Organization Site** instructions, but I have not verified this.

**Comment by nguyenalex836** [comment]
@hey-august Thank you for raising this issue! I'll get this triaged for review :sparkles: Our team will provide feedback regarding the best next steps for this issue - thanks for your patience! 💛

**Comment by Sithusoelwin35** [comment]
05997e97a3e9343d8b57d893cceeeeff31eff08512936c2998c80a80e9c08b4623

**Comment by hey-august** [comment]
@nguyenalex836 Awesome, thanks 😄 
I'd be happy to open a PR to update the docs. just need more information about the state of the custom domain verification feature

**Comment by cinderblock** [comment]
I was also confused by this issue. I was expecting a way to verify a domain on the project that has the pages turned on. I didn't realize it was pushing us to the user/org settings specifically. Maybe a link to those settings could be added to the project's pages.

**Comment by nguyenalex836** [comment]
> Maybe a link to those settings could be added to the project's pages.

@cinderblock Howdy! 👋 Given the age of this issue, would you be willing to open another issue or PR for this proposed update? 💛
